scheduler/scheduler.go

```diff
@@ -366,9 +395,14 @@ func containerCapacity(numCurrentParents, numCurrentContainers, numContainersToS
 // createParents creates host intent documents for each parent
 func createParents(parent distro.Distro, numNewParents int, pool *evergreen.ContainerPool) []host.Host {
 	hostsSpawned := make([]host.Host, numNewParents)
-
+	options, err := generateParentHostOptions(parent.ProviderSettings, pool)
+	grip.DebugWhen(err != nil, message.Fields{
```

Comment:
you can use message.WrapError(err, <message>) rather than the when, and this would also let you omit line 402.